json/src/main/java/com/basistech/rosette/dm/jackson/KnownAttribute.java

```diff
@@ -56,9 +59,7 @@
     TOPIC_RESULT("topicResults", CategorizerResult.class),
     LIST("list", ListAttribute.class),
     UNKNOWN("unknown", UnknownAttribute.class),
-    RELATION_ARGUMENT("RelationshipComponent", RelationshipComponent.class),
-    DEPENDENCY("dependency", Dependency.class);
-
+    RELATION_ARGUMENT("RelationshipComponent", RelationshipComponent.class);
```

Comment:
I know these enums don't matter much but RELATION_ARGUMENT was an incorrect name to begin with. `argument!=component`